src/models/searchservice/response/DirectAnswer.ts

```diff
@@ -1,23 +1,61 @@
 import { Result } from './Result';
+import { DirectAnswerType } from './DirectAnswerType';
+import { Snippet } from './Snippet';
 
 /**
  * A direct answer to a search.
  *
  * @public
  */
 export interface DirectAnswer {
+  /** The {@link DirectAnswerType}. */
+  type: DirectAnswerType | string;
   /** The result of the direct answer. */
   value: string;
   /** The entity associated with the direct answer. */
   relatedResult: Result;
   /** The vertical key of the direct answer. */
   verticalKey: string;
-  /** The name of the entity of the direct answer. */
+}
+
+/**
+ * A direct answer which was found within a document.
+ *
+ * @public
+ */
+export interface FeaturedSnippetDirectAnswer extends DirectAnswer {
```

Comment:
Can we move `FeaturedSnippetDirectAnswer` and `FieldValueDirectAnswer` to their own files?

Reply:
sure